service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/CloneAllResourcesFlight.java

```diff
@@ -114,7 +114,7 @@ private void addFlightLaunchStepsForResource(
                     resourceCloneInputs.getFlightId()),
                 RetryRules.shortDatabase());
           }
-          case CONTROLLED_GCP_AI_NOTEBOOK_INSTANCE ->
+          default ->
```

Comment:
nit: I would either leave `case CONTROLLED_GCP_AI_NOTEBOOK_INSTANCE` directly above default (so it falls through) or else leave a comment here that we intentionally don't support cloning notebooks at the moment.

Reply:
Adding comment, since it is not needed for AWS notebook as well

Reply:
you can also do case CONTROLLED_GCP_AI_NOTEBOOK_INSTANCE, CONTROLLED_AWS_SAGEMAKER_NOTEBOOK_INSTANCE ->

Reply:
Adding a default case is preferred (with comment).
reason: 
the default case applies to all existing (and new resources) that don't explicitly have clone enabled. 
This way we don't miss out on changes for new resources.

Reply:
There are more resources that dont have clone set up, hence default makes more sense than listing all others

Reply:
Ignore previous replies. added separate cases with specific logs